project/MicroServiceBuild.scala

```diff
@@ -57,12 +59,14 @@ private object AppDependencies {
       override lazy val test = Seq(
         "uk.gov.hmrc" %% "bootstrap-backend-play-28" % bootstrapBackendVersion % scope,
         "org.scalatest" %% "scalatest" % scalaTestVersion % scope,
+        "org.scalatestplus" %% "mockito-3-12" % "3.2.10.0" % scope,
         "org.pegdown" % "pegdown" % pegdownVersion % scope,
-        "com.typesafe.play" %% "play-test" % PlayVersion.current % scope,
+        "com.typesafe.play" %% "play-test" % playVersion % scope,
         "org.scalatestplus.play" %% "scalatestplus-play" % scalaTestPlusVersion % scope,
+        "com.vladsch.flexmark" % "flexmark-all" % "0.64.0" % scope,
```

Comment:
`0.64.0` is not compatible with Java 8 and it might be good to extract the version in a variable.